in_test.go

```diff
@@ -266,7 +349,7 @@ func createTestPR(count int, baseName string, skipCI bool, isCrossRepo bool) *re
 		m = "[skip ci]" + m
 	}
 
-	return &resource.PullRequest{
+	retpr := resource.PullRequest{
```

Comment:
Please revert this change as well so there is no diff, since there are no longer any code changes (and no reason to declare `reptr` instead of just returning the PR object directly.

Reply:
done